features/machine/machine.feature

```diff
@@ -638,9 +638,19 @@ Feature: Machine features testing
       | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-cluster"]    | <%= infrastructure("cluster").infra_name %> |
       | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-machineset"] | disconnected-azure-36489                    |
       | ["spec"]["template"]["spec"]["providerSpec"]["value"]["publicIP"]                         | true                                        |
-
+    
+    And admin ensures "disconnected-azure-36489" machine_set_machine_openshift_io is deleted after scenario
+    Then I store the last machine name in the :machine_latest clipboard 
```

Comment:
@miyadav we have similar step `I store the last provisioned machine in the :machine clipboard`

Reply:
Yes @sunzhaohua2 , but it checks for `Provisioned` machine , our machine will go from `provisioning` to `failed`